xbmc/windowing/WinSystem.cpp

```diff
@@ -273,3 +274,27 @@ KODI::WINDOWING::COSScreenSaverManager* CWinSystemBase::GetOSScreenSaver()
 
   return m_screenSaverManager.get();
 }
+
+void CWinSystemBase::RegisterRenderLoop(IRenderLoop *client)
+{
+  CSingleLock lock(m_renderLoopSection);
+  m_renderLoopClients.push_back(client);
+}
+
+void CWinSystemBase::UnegisterRenderLoop(IRenderLoop *client)
+{
+  CSingleLock lock(m_renderLoopSection);
+  std::vector<IRenderLoop*>::iterator i = find(m_renderLoopClients.begin(), m_renderLoopClients.end(), client);
```

Comment:
You can use `auto` to make the end of the line better readable on smaller screens

Reply:
done